interface/portrait_properties/00_portrait_properties.txt

```diff
@@ -4752,4 +4752,34 @@
 			character = 42086
 		}
 	}
+}
+# Warcraft
+# p27 demons
+27 = {
+	0 = {
+		factor = 1
+		trigger = {
+			NOT = {
+				has_portrait_property = { layer = 27 index = 1 }
+				has_portrait_property = { layer = 27 index = 2 }
+			}
+		}
+	}
+	1 = {		# Archimonde
+		factor = 100
+		trigger = {
+			has_game_rule = { name = static_portraits value = enabled }
+			
+			has_character_flag = is_archimonde
```

Comment:
Dead characters lose flags, so portrait may disappear. Suggest using hidden traits instead.
https://ck2.paradoxwikis.com/trait%20modding

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/71586952-5bb3ea00-2b35-11ea-8103-0962c1c866cf.png)
![image](https://user-images.githubusercontent.com/46384992/71586961-64a4bb80-2b35-11ea-9f55-650a795b6ad4.png)

</details>

Reply:
fixed